### PR TITLE
@zamiang More options on video component

### PR DESCRIPTION
**Pull request by kanaabe**
video

### CODE REVIEW

**Comment by zamiang** [comment]
:sparkles: :tada: :sparkles: :

File: client/apps/edit/components/section_tool/index.coffee [hunk 1]
```diff
@@ -41,6 +41,8 @@ module.exports = React.createClass
         @props.sections.add {
           type: 'video'
           url: ''
+          background_color: ''
+          layout: 'column_width'
```
Comment (zamiang) [inline client/apps/edit/components/section_tool/index.coffee]: So `column_width` is the new default - what is the new fullscreen one called?

Comment (zamiang) [inline client/apps/edit/components/section_tool/index.coffee]: Ah! Looks like `overflow_fillwidth`

Comment (kanaabe) [inline client/apps/edit/components/section_tool/index.coffee]: Indeed, we use it in at least one other place too, I think artwork sections

**Comment by zamiang** [comment]
Cool! Good to merge and deploy :+1:

**Pull request merged**